Makefile

```diff
@@ -214,7 +214,9 @@ jaeger-ui/packages/jaeger-ui/build/index.html:
 
 .PHONY: rebuild-ui
 rebuild-ui:
-	cd jaeger-ui && yarn install --frozen-lockfile && cd packages/jaeger-ui && yarn build
+	cd scripts && ./rebuild-ui.sh
```

Comment:
```suggestion
	./scripts/rebuild-ui.sh
```

Reply:
Yeah -C flag was creating packages/jaeger-ui/build/ again. Have fixed this with the latest commit